### ISSUE TITLE
[Bug Report][2.2.11] v-menu on screen reader is not read as a drop-down-list

### ISSUE THREAD

**Issue by ncsugrad94** [vuetifyjs/vuetify#15538]
### Environment
**Vuetify Version:** 2.2.11
**Vue Version:** 2.6.11
**Browsers:** Chrome 103.0.0.0
**OS:** Windows 10

### Steps to reproduce
navigate to the option menu with keyboard. The screen reader should read that it is a drop down menu

### Expected Behavior
screen reader should read the menu as "drop down list or menu'

### Actual Behavior
screen reader does not read it as menu

### Reproduction Link
<a href="https://codepen.io/anzuj/pen/OJyaRLR" target="_blank">https://codepen.io/anzuj/pen/OJyaRLR</a>

**Comment by ncsugrad94** [comment]
also another example where keyboard navigation is not working and screen reader is not functioning properly

https://codepen.io/pen?&editors=101

**Comment by KaelWD** [comment]
- 2.2.11 was **over two years ago**
- If you use it the way [the current documentation](https://vuetifyjs.com/en/components/menus/) says to then you do get the appropriate `aria-haspopup` attribute
